examples/double-conversion/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.0)
+
+include("../common.cmake")
+
+project(download-double-conversion)
+
+hunter_add_package(double-conversion)
+
+# now double-conversion can be used
+find_package(double-conversion CONFIG REQUIRED)
+include_directories(${double-conversion_INCLUDE_DIRS})
```

Comment:
It should be part of imported target.

Reply:
Ya I was just double checking it wasn't ideal. Will keep researching.